pages/index.js

```diff
@@ -77,9 +65,10 @@ export default function Home() {
                         </p>
                     </div>
                 </section>
-                <svg className="transform rotate-180 -mt-1 md:-mt-5" viewBox="0 0 1440 320">
+                <svg
+                    className="fill-current text-blue dark:text-blue-dark transform rotate-180 -mt-1 md:-mt-5"
+                    viewBox="0 0 1440 320">
                     <motion.path
-                        fill="#c1e8ed"
```

Comment:
wow this is smart i don't know why i didn't think of this

it's funny because i told @eilla1 to do this on her repo but i didn't do it myself